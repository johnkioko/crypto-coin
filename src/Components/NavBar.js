import React from 'react';
import 'font-awesome/css/font-awesome.min.css';
// import logo from '../images/logo_crypto.png';

const NavBar = () => (
  <header className="header flex">
    <div className="logo-header flex">
      <div className="logo flex">
        <span className="back">
          <span className="back-txt white">
            <h1 className="logo">CRYPTO</h1>
          </span>
        </span>
      </div>
    </div>
    <div className="right-side flex">
      <span className="audio">
        <i className="fa fa-microphone" aria-hidden="true" />
      </span>
      <span className="settings">
        <i className="fa fa-cog" aria-hidden="true" />
      </span>
    </div>
  </header>
);

export default NavBar;
