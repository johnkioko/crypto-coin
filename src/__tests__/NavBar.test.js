import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import NavBar from '../Components/NavBar';

test('renders the navbar component', () => {
  const { asFragment } = render(
    <NavBar />,
  );

  expect(asFragment()).toMatchSnapshot();
});
