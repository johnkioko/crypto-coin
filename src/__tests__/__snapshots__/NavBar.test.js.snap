// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the navbar component 1`] = `
<DocumentFragment>
  <header
    class="header flex"
  >
    <div
      class="logo-header flex"
    >
      <div
        class="logo flex"
      >
        <span
          class="back"
        >
          <span
            class="back-txt white"
          >
            <h1
              class="logo"
            >
              CRYPTO
            </h1>
          </span>
        </span>
      </div>
    </div>
    <div
      class="right-side flex"
    >
      <span
        class="audio"
      >
        <i
          aria-hidden="true"
          class="fa fa-microphone"
        />
      </span>
      <span
        class="settings"
      >
        <i
          aria-hidden="true"
          class="fa fa-cog"
        />
      </span>
    </div>
  </header>
</DocumentFragment>
`;
